src/Linux/Avalonia.LinuxFramebuffer/Input/LibInput/LibInputBackend.KeyboardMap.cs

```diff
@@ -0,0 +1,192 @@
+using System.Collections.Generic;
+using System.Linq;
+using Avalonia.Input;
+
+namespace Avalonia.LinuxFramebuffer.Input.LibInput
+{
+    public partial class LibInputBackend
+    {
+
+        private static readonly ILookup<LibInputKeyCodes, Key> KeyLookup;
+        
+        private static readonly Dictionary<Key, LibInputKeyCodes> KeyMap = new Dictionary<Key, LibInputKeyCodes>()
```

Comment:
Multiple evdev keys might be mapped to a single avalonia key (e. g. `KEY_KP4` might be `Key.D4` or `Key.Left` depending on the numlock state). We should use libxkbcommon instead of direct mapping.